Layouts/layout-details-Vulnerability_upToV4.json

```diff
@@ -0,0 +1,417 @@
+{
+	"typeId": "Vulnerability",
+	"kind": "details",
+	"releaseNotes": "-",
+	"layout": {
+		"id": "Vulnerability",
+		"version": -1,
+		"modified": "2018-05-23T08:26:33.071254532Z",
+		"name": "",
+		"sections": [
+			{
+				"id": "",
+				"version": 0,
+				"modified": "0001-01-01T00:00:00Z",
+				"name": "Vulnerability Information",
+				"type": "",
+				"isVisible": true,
+				"readOnly": false,
+				"fields": [
+					{
+						"id": "",
+						"version": 0,
+						"modified": "0001-01-01T00:00:00Z",
+						"fieldId": "incident_assetid",
```

Comment:
@Nitzperetz I believe those fields should also appear in `incidentfields.json`
otherwise Demisto will ignore them 
(relevant for all new fields in this layout)